common/exception/ErrorMessage.java

```diff
@@ -47,7 +47,7 @@ private ErrorMessage(String codePrefix, int codeNumber, String messagePrefix, St
         public static final Client MISSING_RESPONSE =
```

Comment:
See comments on https://github.com/vaticle/typedb-client-python/pull/250/files, which this PR is a copy of.